app/interactors/ratingWeets.js

```diff
@@ -0,0 +1,57 @@
+const { Rating, User, sequelize } = require('../models');
+const logger = require('../logger');
+const errors = require('../errors');
+const utilities = require('../helpers/utilities');
+
+const createRatingWeet = async (ragintgWeetData, weetUserId) => {
+  let transaction = {};
+  let rateWeetResult = {};
+  const { ratingUserId, weetId, score } = ragintgWeetData;
+  try {
+    transaction = await sequelize.transaction();
+
+    logger.info(
+      `ratingWeets-interactor::getting rating weet to update::ratingUserId::${ratingUserId}::weetId::${weetId}`
+    );
+    const ratingWeetToUpdate = await Rating.findOne({
+      where: { ratingUserId, weetId },
+      attributes: ['id', 'ratingUserId', 'weetId', 'score']
+    });
+
+    if (ratingWeetToUpdate) {
+      logger.info(
+        `ratingWeets-interactor::updating rating weet::ratingWeetToUpdate::${JSON.stringify(
+          ratingWeetToUpdate
+        )}`
+      );
+      ratingWeetToUpdate.score = score;
+      rateWeetResult = await ratingWeetToUpdate.save({ transaction });
+    } else {
+      logger.info(
+        `ratingWeets-interactor::creating rating weet::ragintgWeetData::${JSON.stringify(ragintgWeetData)}`
+      );
+      rateWeetResult = await Rating.create(ragintgWeetData, { transaction });
+    }
+
```

Comment:
Also, you can use Sequelize.upsert to make the code simpler, instead of checking if the rating already exists. It's important to have an unique index when using upsert, since Sequelize will check the existence by the defined constraints.